### PR TITLE
Gift cards feature migration

### PR DESCRIPTION
**Pull request by gibmirmilch**
## PR Description

TBD

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/scos/dev/feature-walkthroughs/201811.0/gift-cards-feature-walkthrough.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: Gift Cards feature walkthrough
+last_updated: Sep 28, 2021
+description: The Gift Cards feature enables you to create a virtual product (a gift card) with a chosen value amount
+template: concept-topic-template
+---
+
+The _Gift Cards_ feature enables you to create a virtual product (a gift card) with a chosen value amount. The purchase of a gift card generates an individual code that is used as a payment method during checkout.
```
Comment (andriitserkovnyi) [inline docs/scos/dev/feature-walkthroughs/201811.0/gift-cards-feature-walkthrough.md:8]: Technically, all the products are virtual. I suggest using "special product type".

File: docs/scos/user/features/201811.0/gift-cards-feature-overview.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  - /v1/docs/en/gift-card-purchase-management
+  - /v1/docs/gift-cards-purchase-redeeming
+  - /v1/docs/en/gift-cards-purchase-redeeming
+---
+
+The _Gift Card_ feature allows you to create a virtual product with a chosen value amount. The purchase of a gift card generates an individual code that can then be used as a payment method during checkout.
+
+A Gift Card is a prepaid certificate entitling its owner to purchase products for the gift card’s value.
+
+When paying for orders with a gift card, the gift cards are considered as an additional payment method. You can use the full balance  or part of the total gift card's value. In the latter case, the remaining value is stored as leftover balance on the original code. You can also use additional gift cards to pay for products. However, gift cards can only be applied for products in the same currency they have been bought.
```
Comment (andriitserkovnyi) [inline docs/scos/user/features/201811.0/gift-cards-feature-overview.md:19]: In the first clause, `gift card` is singular while in the second one it's plural. Unify it

Comment (andriitserkovnyi) [reply docs/scos/user/features/201811.0/gift-cards-feature-overview.md:19]: `a leftover balance`

Comment (andriitserkovnyi) [reply docs/scos/user/features/201811.0/gift-cards-feature-overview.md:19]: The meaning of the `original code` is not clear. Do you mean the used gift card?

Comment (andriitserkovnyi) [reply docs/scos/user/features/201811.0/gift-cards-feature-overview.md:19]:
```suggestion
You can also use multiple gift cards to pay for a product.
```

**Review by lenadoc — APPROVED** [review]

**Pull request merged**